service/src/test/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneResourcesFlightIntegrationTest.java

```diff
@@ -129,7 +129,10 @@ void setup() {
             .id(UUID.randomUUID());
     landingZoneManager =
         LandingZoneManager.createLandingZoneManager(
-            tokenCredential, azureProfile, resourceGroup.name());
+            tokenCredential,
+            azureProfile,
+            resourceGroup.name(),
+            null /*ignore this value in test*/);
```

Comment:
Is there any meaningful way we can test that the attribute is being wired up and emitted as expected?

Reply:
I see it as unit test to verify if specific method was invoked. Although it's not very convenient now when we have all static configuration. Not sure that we need to do some integration testing in this case.